fireapi/firedb.py

```diff
@@ -60,49 +64,74 @@ def soeg_geometriobjekt(self, bbox) -> List[GeometriObjekt]:
             .all()
         )
 
-    def hent_observationer_naer_punkt(
-        self, punkt: Punkt, afstand: float, tidfra: datetime, tidtil: datetime
+    def __filter(self, g1, g2, distance: float, from_date: Optional[datetime] = None, to_date: Optional[datetime] = None):
+        exps = [
+            func.sdo_within_distance(
+                g1, g2, "distance=" + str(distance) + " unit=meter"
+            )
+            == "TRUE"
+        ]
+        if from_date:
+            exps.append(Observation.observationstidspunkt >= from_date)
```

Comment:
Hmm, I wonder if we are supposed to filter on observationstidspunkt or registrering_fra and registrering_til?

Reply:
Good question, I asked myself the same and chose the one I think is most likely relevant.